### PR TITLE
Add linux aarch64 wheel and fix release name

### PR DESCRIPTION
**Pull request by Thoralf-M**
Tested here https://github.com/Thoralf-M/wallet.rs/actions/runs/4280579030/jobs/7452529593

### CODE REVIEW

File: .github/workflows/release-python-bindings.yml [hunk 1]
```diff
@@ -96,26 +96,65 @@ jobs:
           name: wheels
           path: dist
 
+  build-wheels-docker:
```
Comment (thibault-martinez) [inline .github/workflows/release-python-bindings.yml]: Can this mention `aarch64`? Saying docker is not really helpful

File: .github/workflows/release-python-bindings.yml [hunk 2]
```diff
@@ -96,26 +96,65 @@ jobs:
...
+            cd root
+            mkdir dist
+
+            apt-get update 
+            apt-get install -y git 
+            apt-get install -y curl 
+            apt install -y build-essential 
+            apt install -y libudev-dev libusb-1.0-0-dev 
+            apt install -y pkg-config 
+            apt install -y clang
```
Comment (thibault-martinez) [inline .github/workflows/release-python-bindings.yml]: Maybe group them in a single command?

File: .github/workflows/release-python-bindings.yml [hunk 3]
```diff
@@ -96,26 +96,59 @@ jobs:
...
+            apt-get update 
+            apt-get install -y git curl build-essential libudev-dev libusb-1.0-0-dev pkg-config clang
+            curl https://sh.rustup.rs -sSf | sh -s -- -y 
+            . "$HOME/.cargo/env" 
+
+            cd wallet/bindings/python 
+            pip install -r requirements-dev.txt 
+            pip install patchelf
+            pip install maturin
+            maturin build --out ../../../dist --release
```
Comment (thibault-martinez) [inline .github/workflows/release-python-bindings.yml:124]: Will dist exist if you don't create it ?

Comment (Thoralf-M) [reply .github/workflows/release-python-bindings.yml:124]: Yes, just tested https://github.com/Thoralf-M/wallet.rs/actions/runs/4280579030/jobs/7452529593

**Review by thibault-martinez — DISMISSED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**